integration/docker/Dockerfile-dev

```diff
@@ -9,86 +9,24 @@
 # See the NOTICE file distributed with this work for information regarding copyright ownership.
 #
 
-# ARG defined before the first FROM can be used in FROM lines
-ARG JAVA_VERSION=8
+FROM alluxio/alluxio:2.9.0-SNAPSHOT
 
-# setup CSI
-# Make sure any changes to CSI installation are made in Dockerfile as well
-FROM golang:1.15.13-alpine AS csi-dev
-ENV GO111MODULE=on
-RUN mkdir -p /alluxio-csi
-COPY ./csi /alluxio-csi
-RUN cd /alluxio-csi && \
-    CGO_ENABLED=0 go build -o /usr/local/bin/alluxio-csi
-
-# We have to do an ADD to put the tarball into extractor, then do a COPY with chown into final
-# ADD then chown in two steps will double the image size
-#   See - https://stackoverflow.com/questions/30085621/why-does-chown-increase-size-of-docker-image
-#       - https://github.com/moby/moby/issues/5505
-#       - https://github.com/moby/moby/issues/6119
-# ADD with chown doesn't chown the files inside tarball
-#   See - https://github.com/moby/moby/issues/35525
-FROM alpine:3.10.2 AS alluxio-extractor
-# Note that downloads for *-SNAPSHOT tarballs are not available.
-ARG ALLUXIO_TARBALL=http://downloads.alluxio.io/downloads/files/2.9.0-SNAPSHOT/alluxio-2.9.0-SNAPSHOT-bin.tar.gz
-# (Alert):It's not recommended to set this Argument to true, unless you know exactly what you are doing
-ARG ENABLE_DYNAMIC_USER=false
-
-ADD ${ALLUXIO_TARBALL} /opt/
-# Remote tarball needs to be untarred. Local tarball is untarred automatically.
-# Use ln -s instead of mv to avoid issues with Centos (see https://github.com/moby/moby/issues/27358)
-RUN cd /opt && \
-    (if ls | grep -q ".tar.gz"; then tar -xzf *.tar.gz && rm *.tar.gz; fi) && \
-    ln -s alluxio-* alluxio
-
-RUN if [ ${ENABLE_DYNAMIC_USER} = "true" ] ; then \
-       chmod -R 777 /opt/* ; \
-    fi
-
-# Configure JAVA_HOME
-FROM centos:7 as build_java8
-ENV JAVA_HOME /usr/lib/jvm/java-1.8.0-openjdk
-
-FROM centos:7 as build_java11
-ENV JAVA_HOME /usr/lib/jvm/java-11-openjdk
-
-FROM build_java${JAVA_VERSION} AS final
-
-ADD dockerfile-common.sh /
-
-# Install libfuse2 and libfuse3. Libfuse2 setup is modified from cheyang/fuse2:ubuntu1604-customize to be applied on centOS
-WORKDIR /
-
-ENV MAX_IDLE_THREADS "64"
+USER root
```

Comment:
Why we use USER root here? didn't find we using USER ROOT in the dockerfile

Reply:
is it by default in Dockerfile before we set it to `USER ${ALLUXIO_UID}`

Reply:
Yes, it's because in Dockerfile we set it to `USER ALLUXIO_UID` at the end